sonar-ws/src/main/protobuf/ws-users.proto

```diff
@@ -107,10 +107,16 @@ message CurrentWsResponse {
   optional Permissions permissions = 10;
   optional bool showOnboardingTutorial = 11;
   optional string avatar = 12;
+  optional Homepage homepage = 13;
 
   message Permissions {
     repeated string global = 1;
   }
+
+  message Homepage {
+    optional string type = 1;
```

Comment:
suggestion: you could use an enum for the Homepage type

Reply:
done